vpc.auto.tfvars.example

```diff
@@ -0,0 +1,26 @@
+## Required values
```

Comment:
Isn't this redundant with `variables.tf`? Managing code (and documentation in 2 places) is a recipe for technical debt.

Reply:
I look at this in forms of inheritance. There are the "sane" defaults defined in the `variables.tf`. There are globals defined in the environment. And then there are *project specific* definitions in the `.tfvar` files. Settings necessarily need to be defined at different levels.